### PR TITLE
Allow signoff and disconnect to pass.

### PR DESCRIPTION
**Pull request by ajonp**
Couldn't get logged off.

### CODE REVIEW

**Comment by ajonp** [comment]
I signed it!

**Comment by goderbauer** [comment]
Hm, looks like the tests are still failing on Travis: https://travis-ci.org/flutter/plugins/jobs/236436185

**Comment by ajonp** [comment]
Bumped version, I think I missed the file that allowed the test to pass.

File: packages/google_sign_in/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.0.3
+
+*Allow Disconnect and Signout, also pass tests by adding null
```
Comment (goderbauer) [inline packages/google_sign_in/CHANGELOG.md]: nit: Space after the `*` and on a new line `* fixed tests`

File: packages/google_sign_in/lib/google_sign_in.dart [hunk 2]
```diff
@@ -128,7 +128,7 @@ class GoogleSignIn {
       );
     }
     await _initialization;
-    Map<String, dynamic> response = await _channel.invokeMethod(method);
+    Map<String, dynamic> response = await _channel.invokeMethod(method,null);
```
Comment (goderbauer) [inline packages/google_sign_in/lib/google_sign_in.dart]: This seems strange. `null` is the default value if nothing is passed, I believe. This shouldn't change anything...

**Comment by pulyaevskiy** [comment]
Just found this issue as well. Thanks for fixing this!

File: packages/google_sign_in/lib/google_sign_in.dart [hunk 3]
```diff
@@ -128,7 +128,7 @@ class GoogleSignIn {
...
     _currentUser = response != null ? new GoogleSignInAccount._(this, response) : null;
```
Comment (pulyaevskiy) [inline packages/google_sign_in/lib/google_sign_in.dart]: @goderbauer , @ajonp 

When testing locally the returned `response` for me is an empty `Map`, so on disconnect it tries to instantiate an account and fails on assertions in the constructor.

Comment (sadilek) [inline packages/google_sign_in/lib/google_sign_in.dart]: I was able to reproduce. This is fixed in #93.

File: packages/google_sign_in/lib/google_sign_in.dart (hunk 2 above)
Comment (sadilek) [inline packages/google_sign_in/lib/google_sign_in.dart]: This is to get Mockito not ignore this call because of the missing optional positional argument. I solved this with a change in the test in #93.

**Comment by goderbauer** [comment]
Closing this in favor of https://github.com/flutter/plugins/pull/93.
